### PR TITLE
Create symlink from customization.js to customization-psij/sdk.js on …

### PR DESCRIPTION
**Pull request by hategan**
…upgrade.

For reference, since this is a one-time operation (pip seems to preserve the link when you upgrade a package), this can also be done manually if needed.

### CODE REVIEW

File: deployment/docker/update-psi-j-testing-service [hunk 1]
```diff
@@ -84,6 +87,11 @@ fi
 
 if [ "$ANSWER" == "y" ]; then
     pip install "$PACKAGE"
+    PACKAGE_LOC=`pip show $PACKAGE_NAME | grep "Location: " | sed 's/Location: //'`
+    LINK_NAME="$PACKAGE_LOC/psij/web/instance/customization.js"
+    if [ ! -a "$LINK_NAME" ]; then
```
Comment (andre-merzky) [inline deployment/docker/update-psi-j-testing-service:92]: `[ -a filename ] ` : test if a  filename exists
`[ ! -a filename ] ` : test if `not` and test if `filename`

In other words: the second `-a` acts as binary operator, i.e., as logical `and`.  I think this is not what you want here?

```
$ touch foo
$ rm -f bar
$ [ -a foo ] && echo 'yes foo'
yes foo
$ [ -a bar ] && echo 'yes bar'
$ [ ! -a foo ] && echo 'no foo'
no foo
$ [ ! -a bar ] && echo 'no bar'
no bar 
```

You probably want `-e` instead.

**Review by andre-merzky — CHANGES_REQUESTED** [review]
Generally looks good to me, but a shell test seems bogus.

File: deployment/docker/update-psi-j-testing-service (hunk 1 above)
Comment (hategan) [reply deployment/docker/update-psi-j-testing-service]: Nice catch. Thanks. Fixed.

**Review by andre-merzky — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**